<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorVolumeWidget</class>
 <widget class="QWidget" name="InspectorVolumeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>207</width>
    <height>140</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutVolume">
     <item>
      <spacer name="horizontalSpacerVolume">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>23</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelVolume">
       <property name="text">
        <string>&lt;a href=&quot;Volume&quot; style=&quot;color: white;&quot;&gt;Volume&lt;/a&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderVolume">
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxVolume">
       <property name="maximum">
        <number>200</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDuration">
     <item>
      <spacer name="horizontalSpacerDuration">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelDuration">
       <property name="text">
        <string>&lt;a href=&quot;Duration&quot; style=&quot;color: white;&quot;&gt;Duration&lt;/a&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxDuration">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTween">
     <item>
      <spacer name="horizontalSpacerTween">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelEasing">
       <property name="text">
        <string>&lt;a href=&quot;Tween&quot; style=&quot;color: white;&quot;&gt;Tween&lt;/a&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTween">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDefer">
     <item>
      <spacer name="horizontalSpacerDefer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>29</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelDefer">
       <property name="text">
        <string>&lt;a href=&quot;Defer&quot; style=&quot;color: white;&quot;&gt;Defer&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxDefer">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerDefer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerVolume">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>labelEasing</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorVolumeWidget</receiver>
   <slot>resetTween(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelVolume</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorVolumeWidget</receiver>
   <slot>resetVolume(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelDuration</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorVolumeWidget</receiver>
   <slot>resetDuration(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderVolume</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorVolumeWidget</receiver>
   <slot>sliderVolumeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxVolume</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorVolumeWidget</receiver>
   <slot>spinBoxVolumeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxDuration</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorVolumeWidget</receiver>
   <slot>durationChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxTween</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>InspectorVolumeWidget</receiver>
   <slot>tweenChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelDefer</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorVolumeWidget</receiver>
   <slot>resetDefer(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>-2</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxDefer</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorVolumeWidget</receiver>
   <slot>deferChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>spinBoxVolumeChanged(int)</slot>
  <slot>resetVolume(QString)</slot>
  <slot>resetTween(QString)</slot>
  <slot>resetDuration(QString)</slot>
  <slot>durationChanged(int)</slot>
  <slot>tweenChanged(QString)</slot>
  <slot>sliderVolumeChanged(int)</slot>
  <slot>resetDefer(QString)</slot>
  <slot>deferChanged(int)</slot>
 </slots>
</ui>
